tests/bootstrap/test_service_loading.py

```diff
@@ -0,0 +1,141 @@
+            ContainerConfigurators.env_vars(
+                {"STRICT_SERVICE_LOADING": "1", "SERVICES": "s3,sqs,sns"}
+            ),
+        ]
+    )
+    running_container = ls_container.start()
+    wait_for_localstack_ready(running_container)
+    url = get_gateway_url(ls_container)
+
+    # check service-status returned by health endpoint
+    response = requests.get(f"{url}/_localstack/health")
+    assert response.ok
+
+    services = response.json().get("services")
+
+    assert services.pop("sqs") == "available"
+    assert services.pop("s3") == "available"
+    assert services.pop("sns") == "available"
+
+    assert services
+    assert all(services.get(key) == "disabled" for key in services.keys())
+
+    # activate sqs service
+    client = aws_client_factory(endpoint_url=url)
+    result = client.sqs.list_queues()
+    assert result
+
+    # verify cloudwatch is not activated
+    with pytest.raises(ClientError) as e:
+        client.cloudwatch.list_metrics()
+
+    e.match(
+        "Service 'cloudwatch' is not enabled. Please check your 'SERVICES' configuration variable."
+    )
+    assert e.value.response["ResponseMetadata"]["HTTPStatusCode"] == 501
+
+    # check status again
+    response = requests.get(f"{url}/_localstack/health")
+    assert response.ok
+
+    services = response.json().get("services")
+
+    # sqs should be running now
+    assert services.get("sqs") == "running"
+    assert services.get("s3") == "available"
+    assert services.get("sns") == "available"
+    assert services.get("cloudwatch") == "disabled"
+
+
+def test_eager_service_loading(
+    container_factory: ContainerFactory,
+    wait_for_localstack_ready,
+    aws_client_factory,
+):
+    ls_container = container_factory(
+        configurators=[
+            ContainerConfigurators.random_container_name,
+            ContainerConfigurators.random_gateway_port,
+            ContainerConfigurators.random_service_port_range(20),
+            ContainerConfigurators.env_vars(
+                {"EAGER_SERVICE_LOADING": "1", "SERVICES": "s3,sqs,sns"}
+            ),
+        ]
+    )
+    running_container = ls_container.start()
+    wait_for_localstack_ready(running_container)
+    url = get_gateway_url(ls_container)
+
+    # check service-status returned by health endpoint
+    response = requests.get(f"{url}/_localstack/health")
+    assert response.ok
+
+    services = response.json().get("services")
+
+    assert services.pop("sqs") == "running"
+    assert services.pop("s3") == "running"
+    assert services.pop("sns") == "running"
+
+    assert services
+    assert all(services.get(key) == "disabled" for key in services.keys())
+
+
+def test_eager_and_strict_service_loading(
+    container_factory: ContainerFactory,
+    wait_for_localstack_ready,
+    aws_client_factory,
+):
+    # this is undocumented behavior, to allow eager loading of specific services while not restricting services loading
```

Comment:
FYI I arrived at this PR because I was specifically trying to figure out how to eager load specific services in a specific order without loading everything or preventing services from ever being enabled. 

My use case is building integration tests that have isolated localstack containers and building/deploying images to ecr for lambda as well as spinning up RDS. I ended up getting tripped up because depending on whether RDS or ECR was started first, ECR might be 4510 or 4511. 

All that is to say, at least one person (me) finds this behavior useful and would appreciate the behavior being documented! 

Looking forward to see this PR land either way.